scripts/node_install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+CMAKE_OPTIONS=${CMAKE_OPTIONS:-"-DCMAKE_BUILD_TYPE=Release -DENABLE_NODE_BINDINGS=On -DENABLE_MASON=On"}
```

Comment:
I mainly did this to be compatible with `node-osrm`. I don't feel strongly about this, vanilla build would be fine for me as well.